package.json

```diff
@@ -167,13 +170,12 @@
     "raf": "^3.4.1",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
-    "react-intl": "^2.9.0",
+    "react-intl": "^5.8.2",
     "regenerator-runtime": "^0.13.5",
     "stylelint": "^10.0.1",
     "stylelint-config-terra": "^3.3.0",
     "terra-collapsible-menu-view": "^6.29.0",
-    "terra-dev-site": "^6.15.0",
-    "terra-enzyme-intl": "^3.3.0",
+    "terra-dev-site": "git+https://github.com/cerner/terra-dev-site.git#react-intl-upgrade",
```

Comment:
You're probably better off removing this dependency for now. When/if we add back terra-dev-site it'll have to be v7 and even then i'll be a weird circular dependency.

Reply:
Removed. I'd mostly had it in there to be able to test out my changes.